package.json

```diff
@@ -38,20 +38,20 @@
   "peerDependencies": {
     "@types/angular": ">=1.5",
     "@types/prop-types": ">=15",
-    "@types/react": ">=16",
-    "@types/react-dom": ">=16",
+    "@types/react": ">=18",
+    "@types/react-dom": ">=18",
     "prop-types": ">=15",
-    "react": ">=15",
-    "react-dom": ">=15"
+    "react": ">=18",
+    "react-dom": ">=18"
   },
   "devDependencies": {
     "@types/angular": "^1.6.54",
     "@types/angular-mocks": "^1.7.0",
     "@types/jasmine": "^3.3.9",
     "@types/jquery": "^3.3.29",
     "@types/prop-types": "^15.7.0",
-    "@types/react": "^16.8.6",
-    "@types/react-dom": "^16.8.2",
+    "@types/react": "^18.2.18",
```

Comment:
In the PR's I've done for the packages The latest versions are @types/react: 18.2.64 and @types/react-dom: 18.2.21 

Worth aligning these one way or the other?

Reply:
Makes no real difference as these are Dev Dependencies and React typings are typically semver aligned.
The important part for consumers, the Peer Dependencies are listed as `>= 18`